src/uct/cuda/cuda_ipc/cuda_ipc_iface.h

```diff
@@ -30,38 +30,32 @@ typedef struct uct_cuda_ipc_iface_config_params {
     double                  overhead;            /* estimated CPU overhead */
 } uct_cuda_ipc_iface_config_params_t;
 
-typedef struct uct_cuda_ipc_iface {
+
+typedef struct {
     uct_cuda_iface_t                   super;
-    ucs_mpool_t                        event_desc;              /* cuda event desc */
-    ucs_queue_head_t                   outstanding_d2d_event_q; /* stream for outstanding d2d */
-    int                                eventfd;                 /* get event notifications */
-    int                                streams_initialized;     /* indicates if stream created */
-    CUcontext                          cuda_context;
-    CUstream                           stream_d2d[UCT_CUDA_IPC_MAX_PEERS];
-                                                                /* per-peer stream */
-    unsigned long                      stream_refcount[UCT_CUDA_IPC_MAX_PEERS];
-                                                                /* per stream outstanding ops */
-    uct_cuda_ipc_iface_config_params_t config;                  /* configurable iface parameters */
+    uct_cuda_ipc_iface_config_params_t config;
 } uct_cuda_ipc_iface_t;
 
 
-typedef struct uct_cuda_ipc_iface_config {
+typedef struct {
     uct_iface_config_t                 super;
     uct_cuda_ipc_iface_config_params_t params;
 } uct_cuda_ipc_iface_config_t;
 
 
-typedef struct uct_cuda_ipc_event_desc {
-    CUevent           event;
-    void              *mapped_addr;
-    unsigned          stream_id;
-    uct_completion_t  *comp;
-    ucs_queue_elem_t  queue;
-    uct_cuda_ipc_ep_t *ep;
-    uintptr_t         d_bptr;
-    pid_t             pid;
+typedef struct {
+    uct_cuda_event_desc_t super;
+    void                  *mapped_addr;
+    unsigned              stream_id;
```

Comment:
i think it can be removed since was used for stream_refcount that was also removed